components/job-orchestration/job_orchestration/executor/compress/compression_task.py

```diff
@@ -77,12 +82,56 @@ def update_job_metadata_and_tags(db_cursor, job_id, table_prefix, tag_ids, archi
     )
 
 
-def make_clp_command(
+def _generate_fs_targets_file(
+    output_file_path: pathlib.Path,
+    paths_to_compress: PathsToCompress,
+) -> None:
+    file_paths = paths_to_compress.file_paths
+    empty_directories = paths_to_compress.empty_directories
+    with open(output_file_path, "w") as file:
+        if len(file_paths) > 0:
```

Comment:
Is this necessary?

Reply:
Was too drunk when I copied and wrote the code